mono/utils/mono-threads-posix.c

```diff
@@ -204,30 +204,33 @@ mono_native_thread_set_name (MonoNativeThreadId tid, const char *name)
 	if (!name) {
 		pthread_setname_np ("");
 	} else {
-		char n [63];
+		size_t len = 63;
+		char n [len];
```

Comment:
Sure, I 'll change it to `alloca`!
Generally, however: isn't `mono-threads-windows.c` used when MSVC is involved? Also the CI windows machines don't really complain about it?